website/content/en/docs/building-operators/golang/testing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Testing your Operator project
+linkTitle: Testing with EnvTest
+description: Learn how to ensure the quality of your Operator project 
+weight: 70
+---
+
+## Overview
+
+The Operator SDK project recommends controller-runtime's [envtest][envtest] because it has a more active contributor community, it has become more mature than Operator SDK's test framework, and it does not require an actual cluster to run tests, which can be a huge benefit in CI scenarios. 
+
+## Using EnvTest
+
+You will see that `controllers/suite_test.go` is created. This file contains boilerplate for executing integration tests using [envtest][envtest] with [ginkgo](https://onsi.github.io/ginkgo/) and [gomega](https://onsi.github.io/gomega/).
```

Comment:
I haven't used env test so I'm not sure how it works, but I think this lead-in could be a bit more descriptive. When will I see that `controllers/suite_test.go` has been created? After running env test?

Reply:
You will see that `controllers/suite_test.go` is created for the controller generated for your project when it is scaffolded.

Reply:
looks good